tests/DocBlock/TypeExpressionTest.php

```diff
@@ -591,12 +591,20 @@ public static function provideAllowsNullCases(): iterable
      */
     public function testSortTypes(string $typesExpression, string $expectResult): void
     {
+        $sortCaseFx = static function (TypeExpression $a, TypeExpression $b): int {
+            return strcasecmp($a->toString(), $b->toString());
+        };
+        $sortCrc32Fx = static function (TypeExpression $a, TypeExpression $b): int {
+            return strcasecmp($a->toString(), $b->toString());
+        };
```

Comment:
```suggestion
```

`$sortCrc32Fx` is the same as `$sortCaseFx`, is it needed here?

Reply:
typo in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/commit/f08feb3a585cb232852b974d53d2d5f11659edb9, good catch!